token/token.go

```diff
@@ -217,3 +219,13 @@ func ReadManagerFromContext(ctx context.Context) (*Manager, error) {
 	tokenManager := tm.(Manager)
 	return &tokenManager, nil
 }
+
+// InjectTokenManager is a middleware responsible for setting up tokenManager in the context for every request.
+func InjectTokenManager(tokenManager Manager) goa.Middleware {
+	return func(h goa.Handler) goa.Handler {
+		return func(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+			ctxWithTM := tokencontext.ContextWithTokenManager(ctx, tokenManager)
```

Comment:
I think it's worth adding in the function comment/doc that the token manager injected via this InjectTokenManger function from common will be available for extraction only via https://github.com/fabric8-services/fabric8-common/blob/master/login/tokencontext/token_context.go#L22

Reply:
done [ade8c7b](https://github.com/fabric8-services/fabric8-common/pull/19/commits/9dd26becaa8c10fa9ab17528c4a802dcdade8c7b)